### DISCUSSION TITLE
Adding numbers to border-black does not take effect

### DISCUSSION THREAD

**Discussion by wode0weiyi** [tailwindlabs/tailwindcss#13932] [Help]
The border-black property cannot have a number appended to it to set transparency, and attempts to do so have been ineffective. However, the border-gray property can have a number appended to adjust transparency, which works as expected. What is the reason for this discrepancy?

**Comment by wongjn** [comment]
Seems to be working in [this Tailwind Play](https://play.tailwindcss.com/4YV4EERXUp), which would suggest there is something in your project causing the unexpected behavior. Consider providing a project that reproduces the issue you are experiencing.
